provider/cmd/pulumi-resource-aws/generate.go

```diff
@@ -12,13 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build ignore
 // +build ignore
```

Comment:
do we want to remove the old syntax here?

Reply:
Looks like the new syntax is supported in 1.16 and later, I'll fix it up and remove +build